src/test/java/org/cactoos/map/MapEnvelopeTest.java

```diff
@@ -46,93 +43,73 @@
 @SuppressWarnings("PMD.TooManyMethods")
 public final class MapEnvelopeTest {
 
-    /**
-     * A rule for handling an exception.
-     */
-    @Rule
-    public final ExpectedException exception = ExpectedException.none();
-
     @Test
     public void putThrowsException() {
-        this.exception.expect(UnsupportedOperationException.class);
-        this.exception.expectMessage(
-            "#put() is not supported, it's a read-only map"
-        );
         MatcherAssert.assertThat(
```

Comment:
@atapin please use `Assertion` instead of the static methods from `MatcherAssert`